store/timer_layer.go

```diff
@@ -568,6 +568,22 @@ func (s *TimerLayerChannelStore) ClearCaches() {
 	return
 }
 
```

Comment:
My execution of `make store-layers` seems to have picked up some unrelated forgotten new methods.